projects/js-packages/components/components/pricing-table/stories/index.tsx

```diff
@@ -0,0 +1,58 @@
+import Button from '../../button';
+import ProductPrice from '../../product-price';
+import { PricingTable, PricingTableColumn, PricingTableHeader, PricingTableItem } from '../index';
```

Comment:
```suggestion
import PricingTable, { PricingTableColumn, PricingTableHeader, PricingTableItem } from '../index';
```

`PricingTable` is default and not named export. Storybook isn't working for this.